src/storage/overflow_file.c

```diff
@@ -522,22 +511,15 @@ overflow_update (THREAD_ENTRY * thread_p, const VFID * ovf_vfid, const VPID * ov
       data += copy_length;
       length -= copy_length;
 
-      if (isnewpage)
-	{
-	  log_append_redo_data (thread_p, RVOVF_NEWPAGE_INSERT, &addr, copy_length + hdr_length, addr.pgptr);
-	}
-      else
-	{
-	  log_append_redo_data (thread_p, RVOVF_PAGE_UPDATE, &addr, copy_length + hdr_length, (char *) addr.pgptr);
-	}
+      log_append_redo_data (thread_p, RVOVF_PAGE_UPDATE, &addr, copy_length + hdr_length, (char *) addr.pgptr);
```

Comment:
Please look at `la_get_recdes`:

```c
  if (*rcvindex == RVOVF_CHANGE_LINK)
    {
      /* if overflow page update */
      error = la_get_overflow_recdes (lrec, logs, recdes, RVOVF_PAGE_UPDATE);
      recdes->type = REC_BIGONE;
    }
  else if (recdes->type == REC_BIGONE)
    {
      /* if overflow page insert */
      error = la_get_overflow_recdes (lrec, logs, recdes, RVOVF_NEWPAGE_INSERT);
    }
  else if (*rcvindex == RVHF_INSERT && recdes->type == REC_ASSIGN_ADDRESS)
    {
      error = la_get_next_update_log (lrec, pg, &logs, &rec_type, &recdes->data, &recdes->length);
      if (error == NO_ERROR)
        {
          recdes->type = *(INT16 *) (rec_type);
          if (recdes->type == REC_BIGONE)
            {
              error = la_get_overflow_recdes (lrec, logs, recdes, RVOVF_NEWPAGE_INSERT);
            }
        }
    }
```

It expects `RVOVF_NEWPAGE_INSERT` log to trace overflow insertion. 
I've thought about replacing it as `RVOVF_PAGE_UPDATE` but I'm not sure. 
Maybe it is better to leave a dummy log record to help log applier.